docxcompose/utils.py

```diff
@@ -26,3 +27,32 @@ def xpath(element, xpath_str):
         return element.xpath(xpath_str)
     else:
         return element.xpath(xpath_str, namespaces=NS)
+
+
+date_format_map = (
+    ('Y', 'y'),  # Upper or lower case Y are equivalent
+    ('yyyy', '%Y'),
+    ('yy', '%y'),
+    ('MMMM', '%B'),
+    ('MMM', '%b'),
+    ('MM', '%m'),
+    ('M', '%m'),  # M should be without leading 0, but there is no equivalent in python
```

Comment:
On Unix `'%-m'` should work. It's not supported on Windows I guess, but who cares.

Reply:
Thanks, I did not realise that there were more formats only supported on certain platforms 👍

Reply:
`strftime` is implemented by the underlying C standard library. On Linux this is either glibc or musl.
Python only lists the C89 standard format codes which is pretty old. To get the supported codes of your current standard library implementation you could lookup the man page of strftime, e.g. https://man7.org/linux/man-pages/man3/strftime.3.html